src/api/controllers/authController.js

```diff
@@ -97,4 +98,190 @@ export default class Auth {
       return sendError(status.SERVER_ERROR, res, 'logout', 'Please try again, Thanks');
     }
   }
+
+  /**
+   * This function gets all the users that are stored in the User table
```

Comment:
@Ramadhan0 can we put this file in an admin namespace? Let the controller name and filename also be something like `adminPermissions` so it is clear this is only accessible to admins.